backend-models/service-tests/accessedservice-tests.txt

```diff
@@ -0,0 +1,129 @@
+
+accessedservice.php
+
+-------------------
+
+********
+* pre-req
+********
+
+(checklogin() is TRUE && (hasAccess($userid, $cid, 'w') || isSuperUser($userid,) is TRUE))
+
+--------------------
+
+Test 1: try to update a created user by changing the username
+Row: 62 - 80 and 220 - 238
+Send {
+   $opt = "ADDUSER"
+   $uid = "UNK"
+   $regstatus = "UNK"
+}
+To make sure the update of a user works and what response it will get form the database.
+It can be done by creating a user with SQL-query.
+   SQL-query: insert into user( username, email, firstname, lastname, ssn, password, addedtime, class) 
+       VALUES ( :username, :email, :firstname, :lastname, :ssn, :password, now(), :class)
+
+Send {
```

Comment:
first of, where are the proper variable values, have you actually tried inserting the values bellow in a query?
you need to have proper tested values that
1. make sense in the context
2. is a valid input

you are aware that these are if else statements? 

![image](https://user-images.githubusercontent.com/102600391/234268393-c15494fb-ab9f-4803-a8dd-a852f3f76d5b.png)

only one of these can execute at a given time. which means that you ether need to make a test for each individual if else statements.
The alternative is to specify what variable values each variant of the test will use, the person writing the test is not suppose to have to figure this out.